src/components/breed-detail.js

```diff
@@ -31,7 +31,16 @@ const BreedDetail = ({ breed }) => {
       <section>
         <Row>
           <Col xs="auto" className="mb-4 me-lg-5">
-            <ImageWrapper width="270" height="270" src={breedImage[0].url} alt={name} />
+            {breedImage.length ? (
+              <ImageWrapper width="270" height="270" src={breedImage[0]?.url} alt={name} />
+            ) : (
+              <ImageWrapper
+                width="270"
+                height="270"
+                src="https://images.unsplash.com/photo-1577349516274-37ff88a53627?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1035&q=80"
+                alt={name}
+              />
+            )}
```

Comment:
Both `ImageWrapper`s are basically the same, why not just specify the appropriate prop?